File: solutions/beecrowd/1038/1038.py
import sys

for line in sys.stdin:
    x, y = map(int, line.split())

    if x == 1:
        print(f'Total: R$ {y * 4.00:.2f}')
    elif x == 2:
        print(f'Total: R$ {y * 4.50:.2f}')
    elif x == 3:
        print(f'Total: R$ {y * 5.00:.2f}')
    elif x == 4:
        print(f'Total: R$ {y * 2.00:.2f}')
    else:
        print(f'Total: R$ {y * 1.50:.2f}')

File: solutions/beecrowd/1038/1038.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[n q] (->> (str/split line #" ")
                       (map #(Integer/parseInt %)))]
        (cond
          (= n 1)
          (printf "Total: R$ %.2f%n", (* 4.0 q))
          (= n 2)
          (printf "Total: R$ %.2f%n", (* 4.5 q))
          (= n 3)
          (printf "Total: R$ %.2f%n", (* 5.0 q))
          (= n 4)
          (printf "Total: R$ %.2f%n", (* 2.0 q))
          :else
          (printf "Total: R$ %.2f%n", (* 1.5 q))))
      (recur (read-line)))))

(main)
